client/actions/assignments/notifications.js

```diff
@@ -43,12 +51,16 @@ const onAssignmentUpdated = (_e, data) => (
             return;
         }
 
-        if (currentDesk === data.assigned_desk || currentDesk === data.original_assigned_desk) {
+        // Updates my assignments count.
+        dispatch(assignments.ui.myAssignments(['assigned', 'submitted']));
```

Comment:
Consider using `ASSIGNMENTS.WORKFLOW_STATE.ASSIGNED` and `ASSIGNMENTS.WORKFLOW_STATE.SUBMITTED` throughout this PR